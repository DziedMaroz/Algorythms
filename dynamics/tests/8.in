4299670
616
766
774990
16
979099
4
9

4
107233
572565
74
323312

9
317892
077
37
120
489
72
13
55744
5437
874

0

07557
85
0048
200


06
8588

618677
447
561653
148230
88460
31013
342815
298
406113

414
6453
7866
5219

291193
43082
5
1121
694313
71654

83507
9360
812320
5911
673
0
164
4
2095
982913
260
828

515
75830
41450
5467
769748
68661
5340
3710
5
952820
179434
737437
116
02218
81
850575
498
0
86
19898
04316
00
5534
2
44
742860
79177
263

656703
172
25983
63
1944
9135
91
5
467
06898
0893
22906

251
254183
980878
2914
028695
8
3379
773445
126605
52427
543
4
79078
025707
75558
3849
6918

31
975747
0
97378

27
46
222795
66842
4559
7184
7196
41545

6170
341368
6762
9605
386
57
2841
6616
04898


0
8280

21241
691533

75
0580
27759
0050
626162
027
6772
4860
9
3
407
52
425
01
78286
07002
67074
88476
6908
912415
83

247

6
0315
885279
8
525
8
91
0426
25767
888
5122
90032
332458
79
58633
0669
9490
302
792
297
752924

213
5
98488
6
390
2
182926
72087
7
018
481359
921396
14
636
0422
4
218978

6081
0156
4
1
435669
6845
282938
201882
11571
301296


079
1
0
06

6096

3
16
56
9239
24
390865
721

695

4377

66000
841158
8

328066
02

4
06816
53000
412
32110
1
6
3

5781
06581

5
87
879
512456
86
44736

281934
0437
80
03811
060
970
18
2

011560
14623
28
055322

7271
910188
99

996
42314
5883
3
033

730
8699
09737
287130
0
2994
48020
351
69259
9135
893
045
90
625
5402
276623

60
79548
0
54
129465

30203
340927
38445
86
1
7852
6568

826
472
51650
6049


9128
28
458217
0434
896

216445
47003
261448
4177
1
253011

94
988468
55524
90188
6750

18348
6205
128538
81363
902080
69536
221878
57120

4

4239
59
646
4525

90

148


2962
271
88461
474
2805
13
3446
801193
390
47
16
69225
394
03285

9
99688
7216
4
87714
77
34
51539
175
1
48795


9125
075
87319
54491
172069

201
39
28
8128
28175
425371
20
0

16
134
0722
73204
2655
8027
36
05623
50330
75
71961

675799

385107
086475
76612
53



86260
3
8

6
13
4
135
33675
901
767
706
8228
794991
76382
16023
184
0
3
952109
8
66
3
4
114

001
8
362
9
7385
052427

8559
8128
3
80
006364
85186
1
194
3362
17
3
4416
545
61744
650
768448
01160

3099
8746
44
2794
312
6
69
84406


9604
443298
34218
410116
60

008341

24

6

148156
31
159


277
5
26448
10585
98416
922800
7
996821
3
28946
543
1565
06

7186
78947
9
424
0868
75
98
38
12126

67784
458
85050
738
514434
94

3
74
665218
0023
5908
4677
113053
88
680120
0669


68
53605
8415
025722
37212

308
789
946256
63

40697

358
0
089864
3
09371
738683
749611
17
29
2007
4
301459

478271
13478
6
096
8962
0
88186
844
46
126
395

59725
205509
26563
3
69233

1017

0271

