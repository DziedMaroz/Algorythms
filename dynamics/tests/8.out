5
4 2 996 7 0
